src/components/Banner/Banner.scss

```diff
@@ -148,3 +150,30 @@ $secondary-action-horizontal-padding: 1.5 * spacing(tight);
   right: 0;
   padding: (spacing(tight) * 1.5);
 }
+
+
+h1 {
+  @include text-style-heading-h1;
+  @include text-style-semi-bold;
+}
+h2 {
+  @include text-style-heading-h2;
+  @include text-style-bold;
+}
+h3 {
+  @include text-style-heading-h3;
+  @include text-style-semi-bold;
+}
+h4 {
+  @include text-style-heading-h4;
+  @include text-style-normal;
+}
+h5 {
+  @include text-style-text-body-large;
+}
+h6 {
+  @include text-style-text-body-mid;
+}
+i {
+  @include text-style-text-body-small;
+}
```

Comment:
How are all these heading styles being used within this component? I only see `h6` usage within Banner.tsx.

Reply:
@Farhan-emgage TODO: Please explain this during our meeting tomorrow.